packages/trader/src/App/Components/Form/CompositeCalendar/composite-calendar-mobile.jsx

```diff
@@ -112,7 +112,23 @@ const CompositeCalendarMobile = React.memo(
             }
 
             if (key === 'to') {
-                setTo(e.target?.value ? toMoment(e.target.value).format('DD MMM YYYY') : '');
+                // Handling future dates input for IOS devices
+                const value = e.target?.value ? toMoment(e.target.value).format('DD MMM YYYY') : '';
+
+                setTo(value);
+
+                if (value) {
+                    const upcoming_days = parseInt(daysFromTodayTo(value));
+                    if (!isNaN(upcoming_days)) {
+                        setTo('');
```

Comment:
Is there a reason we set this to `''` first and then set a value again down below?

Reply:
This is a trick to force the datepicker to change its value on ios devices. Apparently without this, picking future date will work fine once, but after another attempt it will accept the future dates and won't force it to current date. @akmal-binary if you have better work around for this. Please let me know, I'm also looking for a cleaner and best way to handle this.

Reply:
@prince-deriv You can use create a ref for the `to datepicker` and you can change the value of the the input datepicker element. In this way, you can avoid unnecessary rerenders.
```
setTo(today);
to_datepicker_ref.current.value = today
```

Reply:
@balakrishna-binary  thanks for this

Reply:
@balakrishna-binary  I instead change the mobile datepicker component. It will be redundant to apply the rule each time we are using that component.